src/PageContainer.js

```diff
@@ -65,12 +80,25 @@ class PageContainer extends Component {
                 {this.state.content === PageContent.CALENDAR && (
                     <div style={{ marginTop: '120px', marginLeft: '250px' }}>
                         Calendar
-                        <EventCard
-                            eventType={DeliveryType.RECURRING}
-                            startTime="10am"
-                            endTime="12pm"
-                            futureEvent={true}
-                        />
+                        {this.state.dialogOpen ? (
+                            <Dialog
+                                closeDialog={this.closeDialog}
+                                eventType={DeliveryType.RECURRING}
+                                accountType={this.props.account.accountType}
+                                date="11/14/2017"
+                                startTime="10am"
+                                endTime="12pm"
```

Comment:
Since John changed the schema to have timestamps, maybe it'll be best to incorporate that into the UI now, thus removing `date="11/14/2017" startTime="10am" endTime="12pm"`  and replacing it with something like `endTimeStamp:"1483232400000"`. Unless you have already decided to have that info passed in from the calendar component, which would be fine too!
![image](https://user-images.githubusercontent.com/17226026/38487481-220b9726-3b95-11e8-888f-0e133e026d04.png)

Reply:
it's true that we changed the schema to be `timestamp`s instead, but the screenshot here is for delivery *requests*, not an actual *delivery*, and we actually don't have a *delivery* object yet in the db. But I actually think it's okay to keep it as is for now and let backend dev change it (it'll prob be faster for a backend dev handle it). Good call though!